site/new_site_sources/templates/about_content.tmpl

```diff
@@ -167,14 +167,17 @@ href="http://www.hamakor.org.il/%D7%A7%D7%94%D7%99%D7%9C%D7%94/%D7%99%D7%95%D7%9
       <h1 id="about_site">About This Site</h1>
       <p>
          The new Israel.pm site is an
-	 <a href="http://html-template.sourceforge.net/">HTML::Template</a> based site,
+	 <a href="https://metacpan.org/release/HTML-Template">HTML::Template</a> based site,
 	 with the look being based on
 	 the <a href="http://www.oswd.org/viewdesign.phtml?id=1192">Gila</a> template, which was
 	 downloaded from the <a href="http://www.oswd.org/">Open Source Web Design</a> site.<br />
-	 The site sources are managed under <a href="http://subversion.tigris.org/">Subversion</a>,
+	 The site sources are managed using <a href="https://github.com/szabgab/perl.org.il">a GitHub repository</a> - see <a href="https://guides.github.com/">their tutorials</a> for how to contribute.
+     </p>
+
+     <tmpl_if removed__subversion>
```

Comment:
Why do we need this removed__subversion param? We are done with subversion we won't go back. You can remove all the relevant info. (Maybe keep a link to "historic" repo")